go.mod

```diff
@@ -1,18 +1,18 @@
 module github.com/mineiros-io/terradoc
 
-go 1.17
+go 1.18
```

Comment:
Does terradoc now require at least go 1.18 ? :thinking:

Reply:
I just added everything I had and committed.. but it's not needed. With this change I was testing if go1.17 still builds..

I'm going to mark this PR as draft.